examples/mysql-parameterized-query/mysql_parameterized_query.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE demonstrates how to use `sql:ParameterizedQuery` to easily create pre compiled SQL statements which can be reused.
+keywords: ballerina, ballerina by example, bbe, mysql, parameterized query, parameters, update, insert, delete
```

Comment:
Shall we remove one keyword? The best practice is to have a maximum of 5-8 keywords.

Reply:
Please see above comment

Reply:
Fixed in ab7ae07